platform/source/mbed_retarget.cpp

```diff
@@ -118,6 +122,7 @@ void mbed_set_unbuffered_stream(std::FILE *_file);
 
 void remove_filehandle(FileHandle *file)
 {
+#if !(MBED_CONF_PLATFORM_STDIO_MINIMAL_CONSOLE_ONLY)
```

Comment:
Cannot be removed because it is used in `FileBase.cpp` unless it is also removed there.
Should be change other files as well or should the changes be only on `mbed_retarget.cpp/h`?

Reply:
The function is declared as extern in `FileBase.cpp`
```
extern void remove_filehandle(FileHandle * file);
```
So you can successfully compile blinky (which does not use FileBase.cpp) with `remove_filehandle` compiled out.

Reply:
[Done](https://github.com/hugueskamba/mbed-os/compare/686fe66feaa50c040384b25636e6a7906a59779d..93f364edd2488439214bd28a928fcff6910377e7).